src/gcinfo/arraylist.cpp

```diff
@@ -4,6 +4,7 @@
 
 #include <stdint.h>
 #include <windows.h>
+#include <ex.h>
```

Comment:
This library is used by the JIT. I am not sure whether `ex.h` works well with the JIT. It may be worth verifying that when the JIT throws this exception, it will actually turn into something diagnosable.

Reply:
It should work, the JIT's own allocator throws exceptions, though it does so by calling `RaiseException` directly. I think I'm more concerned about adding an extra dependency on utilcode to the JIT - currently JIT Release builds have very few (and questionable) dependencies that drag in some 30-50 kbytes of code.